projects/hip/docs/install/build.rst

```diff
@@ -43,6 +43,13 @@ Set the repository branch using the variable: ``ROCM_BRANCH``. For example, for
 
    export ROCM_BRANCH=rocm-6.1.x
 
+Starting in ROCM 7.1 release, HIP will be a part of core rocm projects resides in a mono-repository,
```

Comment:
I don't understand the need for the rocm-6.1.x branch above if this is for the 7.1 version of HIP. It seems like the instructions would be for the current release, and prior instructions could be accessed from prior releases?